app.py

```
@@ -0,0 +1,313 @@
+from fasthtml.common import *
```

Ruff (0.14.1)

`from fasthtml.common import *` used; unable to detect undefined names

(F403)